samples/v1.2/Elements/Input.Text.InlineAction.json

```diff
@@ -3,26 +3,19 @@
 	"type": "AdaptiveCard",
 	"version": "1.2",
 	"body": [
-		{
-			"type": "TextBlock",
-			"text": "Text input with an inline action"
-		},
 		{
 			"type": "Input.Text",
 			"id": "iconInlineActionId",
+			"label": "Text input with an inline action",
 			"inlineAction": {
 				"type": "Action.Submit",
 				"iconUrl": "https://adaptivecards.io/content/send.png",
-				"title": "Send"
+				"tooltip": "Send"
```

Comment:
Are we ok with mixing higher versions of features than the version number on the sample's folder implies?

Reply:
I wondered the same and it looks like that's the path we're going to follow

https://github.com/microsoft/AdaptiveCards/pull/6453#discussion_r719797898